UnityProject/Assets/Scripts/Editor/Tools/Networking.cs

```diff
@@ -87,7 +87,7 @@ private static void KillLocalPlayer()
 	{
 		if (CustomNetworkManager.Instance._isServer)
 		{
-			PlayerManager.LocalPlayerScript.playerHealth.Death();
+			PlayerManager.LocalPlayerScript.playerHealth.ApplyDamage(null, 99999f, AttackType.Internal, DamageType.Brute);
```

Comment:
So is Death() the wrong way of killing creatures?
If so, maybe it should be made private

Reply:
Good point, will do